benchmarks/src/main/kotlin/com/google/samples/apps/nowinandroid/foryou/ForYouActions.kt

```diff
@@ -25,6 +26,8 @@ import com.google.samples.apps.nowinandroid.waitAndFindObject
 import com.google.samples.apps.nowinandroid.waitForObjectOnTopAppBar
 import org.junit.Assert.fail
 
+private const val TAG = "ForYouActions"
+
 fun MacrobenchmarkScope.forYouWaitForContent() {
     // Wait until content is loaded by checking if topics are loaded
     device.wait(Until.gone(By.res("loadingWheel")), 5_000)
```

Comment:
In the past there was an issue sometimes that workmanager would sleep for ~20 seconds when you kill the app process too often, which was happening for `CompilationMode.Partial()`. Maybe we need to increase the timeout here?